<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyDC Dragon Code Translator</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonFollowStyle</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="DCTextBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>771</width>
      <height>141</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Dragon Code goes here&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="plainText">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Paste Dragon Code here</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decodeButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Decodeâ†“</string>
    </property>
   </widget>
   <widget class="QPushButton" name="encodeButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Encodeâ†‘</string>
    </property>
   </widget>
   <widget class="QFrame" name="decodedFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>771</width>
      <height>151</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="species">
     <property name="geometry">
      <rect>
       <x>154</x>
       <y>10</y>
       <width>148</width>
       <height>27</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Species</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
     <item>
      <property name="text">
       <string>Dragons</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Humanoids</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Amphibians</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Birds</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Crustaceans</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dinosaurs</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Extra-Terrestrials</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fish</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Insects</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Legends</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mammals</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Molluscs</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mythical</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Plants</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Reptiles</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Spirits</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Undead</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Shape Changers</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Unknown</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Multiple</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="versionLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>DC version</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>48</width>
       <height>27</height>
      </rect>
     </property>
     <property name="wrapping">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="correctionMode">
      <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>2</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
    <widget class="QComboBox" name="subSpecies">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>149</width>
       <height>27</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Subspecies</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
     <item>
      <property name="text">
       <string>Amphiteres</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Draconids</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dragonettes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Eastern Dragons</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Faerie Dragons</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hydra</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dimar</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dracolich</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pernese Dragons</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Turtle Dragons</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Serpents</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tarrasques</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pseudodragons</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Wyverns</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Western Dragons</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Wyrms</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="subSubSpecies">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>10</y>
       <width>121</width>
       <height>27</height>
      </rect>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
     <item>
      <property name="text">
       <string>Sea serpents</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fire serpents</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPushButton" name="copyButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy ðŸ“‹</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pasteButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Paste ðŸ“‹</string>
    </property>
   </widget>
  </widget>
  <action name="actionTest_Menu_Item_One">
   <property name="text">
    <string>Test Menu Item One</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
